wrappers/java/build.gradle

```diff
@@ -1,11 +1,44 @@
-plugins {
-    id 'java'
-}
+apply plugin: 'java'
+apply plugin: 'application'
 
 version '1.0-SNAPSHOT'
 
 sourceCompatibility = 1.8
 
+mainClassName = 'tech.pegasys.era.tools.EraNetworkConfig'
+
+// build extra start script for EraViewer
+task buildEraViewerScript(type: CreateStartScripts) {
+    description "Creates OS specific scripts to call the 'other' entry point"
+    classpath = startScripts.classpath
+    outputDir = startScripts.outputDir
+    mainClassName = 'tech.pegasys.era.tools.EraViewer'
+    applicationName = 'eraviewer'
+}
+
+// build extra start script for EraDeploy
+task buildEraDeployScript(type: CreateStartScripts) {
+    description "Creates OS specific scripts to call the 'other' entry point"
+    classpath = startScripts.classpath
+    outputDir = startScripts.outputDir
+    mainClassName = 'tech.pegasys.era.tools.EraDeploy'
+    applicationName = 'eradeploy'
+}
+
+// build start scripts
+// Note: This causes double overwrite during unzip of poc.zip and -o option is
+//       therefore required. This approach is temporary and good enough for now.
+distZip {
```

Comment:
distZip adds the result of the mentioned tasks to the zip distribution file, but not to the other distribution files (jar, tar). This means that those other files are incorrect and/or unnecessary. Please either fix them or disable them.

Reply:
Particularly, please make sure that "./gradlew installDist" works as expected.